src/routes/app/[id]/+page.server.ts

```diff
@@ -3,51 +3,77 @@ import prisma from '$lib/prisma';
+                
+                        <p style="line-height: 2;">
+                            Thanks for typing in your email! 
+                
+                         <br /> Click the link below to confirm your email address:
+                        </p>
+                
+                        <a href="#" style="background-color: #0077b6; color: white; padding: 15px 32px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 20px 0; cursor: pointer; border-radius: 4px;">
+                            Verify Now
+                        </a>
+                
+                
+                        <p>
+                            This verification link will expire in 24 hours. -From Team SIL
+                        </p>
+                    </div>
+                
+                </body>
+                </html>`
+            };
 
-    transporter.sendMail({
-        from: EMAIL_FROM, // sender address
-        to: ['usercontenttest@gmail.com'], // list of receivers
-        subject: 'Email Verification', // Subject line
-        text: 'Click the link below to verify your email:', // plain text body
-        html: `<!DOCTYPE html>
-        <html>
-        <head>
-            <title>Email Confirmation</title>
-        </head>
-        <body style="font-family: Arial, sans-serif; text-align: center;">
-        
-            <div style="max-width: 600px; margin: auto; padding: 20px;">
-                <h1 style="font-size: 48px; color: #0077b6;">Verification</h1>
-        
-                <p style="font-size: 24px;">Hello [email address],</p>
-        
-                <p style="line-height: 2;">
-                    Thanks for typing in your email! 
-        
-                 <br /> Click the link below to confirm your email address:
-                </p>
-        
-                <a href="#" style="background-color: #0077b6; color: white; padding: 15px 32px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 20px 0; cursor: pointer; border-radius: 4px;">
-                    Verify Now
-                </a>
-        
-        
-                <p>
-                    This verification link will expire in 24 hours. -From Team SIL
-                </p>
-            </div>
-        
-        </body>
-        </html>`
-    });
+            const sendEmail = async (message: Options) => {
+                await new Promise((resolve, reject) => {
+                    transporter.sendMail(message, (err, info) => {
+                        if (err) {
+                            console.error(err);
+                            reject(err);
+                        } else {
+                            resolve(info);
+                        }
+                    });
+                });
+            };
 
-    return { app };
-}) satisfies PageServerLoad;
+            await sendEmail(message);
+            return {
+                success: 'Email is sent'
```

Comment:
Look at how an Action is supposed to return value.

https://kit.svelte.dev/docs/form-actions#anatomy-of-an-action